src/request.js

```diff
@@ -100,12 +100,13 @@ function handleMessage(rawMessage, response) {
 	if (!observerObj) throw new Error('No associated request for the server message', rawMessage);
 	else {
 		let observer = observerObj.observer;
+		const request = observerObj.request;
 		response.body.__header = response.header;
 
 		clearTimeout(observerObj.timeout);
 
 		if (response.header.statusCode !== 200) {
-			observer.onError(response.body);
+			observer.onError(response.body, request);
```

Comment:
I thought I was testing it and that it was working, but it turns out that where I thought rxws was `onError`ing, it really wasn't. So my testing was for naught.

Do you think it makes sense to `onError` an object that looks like `{err: ..., request: {...}}`? Or are there already too many request middlewares depending on the old API?